index.html

```diff
@@ -73,18 +65,18 @@ <h2>What we do</h2>
         <h2>Getting started</h2>
         <h4>Using the Driver</h4>
         <p>
-            Build a tracker, a controller, an HMD, or anything else your heart desires, either using one of our guides or by coming up with something new. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
+            Build a tracker, a controller, an HMD, or anything else your heart desires, either using whatever hardware your heart desires. With hobo_vr you can add anything to SteamVR. You can find more info on how to get started <a href="/docs/html/">here</a>.
         </p>
 
         <h4>Contributing</h4>
         <p>
-            The best way to get in touch with us is through our <a href="https://github.com/HoboVR-Labs">GitHub organization</a>.
-            <br>We'll be happy to review your pull requests or patches.
+            A good place to start is our <a href="https://github.com/HoboVR-Labs/issue_tracker/issues">issue tracker</a>, there you can find feature requests, bug reports, archives, conventions and other issues. It's a good place to start.
```

Comment:
Said 'good place to start' at both the start and end of this paragraph, which feels redundant.

Reply:
I should probably stop writing stuff when im 4 hours of sleep x)

Reply:
For good measure:
`A good place to start is our <a href="https://github.com/HoboVR-Labs/issue_tracker/issues">issue tracker</a>, there you can find feature requests, bug reports, archives, conventions, and other issues.`